compiler/grainformat/reformat.re

```diff
@@ -1829,14 +1829,16 @@ and print_arg_lambda =
                 ~remove_comments=after_brace_comments,
                 comments,
               );
+
+            let print_attribute = (expr: Parsetree.expression) =>
+              print_attributes(expr.pexp_attributes);
             let printed_expressions =
               block_item_iterator(
                 ~previous=Block(expression.pexp_loc),
                 ~get_loc,
                 ~print_item,
                 ~comments=cleaned_comments,
-                ~get_attribute_text=
-                  expr => print_attributes(expr.pexp_attributes),
+                ~get_attribute_text=print_attribute,
```

Comment:
ooooh, as a cleanup, can we name this `print_attribute`? To me that is much more clear 🤔 What do you think?